src/qml/Mysthea/Components/MainDrawer.qml

```diff
@@ -0,0 +1,344 @@
+            id: rulebookDelegate
+            width: menuDrawer.width
+            height: 64
+            topPadding: 21
+            bottomPadding: 21
+            text: qsTr("Rulebook")
+            Material.foreground: Palette.silverChalice
+            icon.source: "qrc:/assets/icons/rulebook.svg"
+            icon.color: Palette.silverChalice
+            onClicked: menuDrawer.rulesbookClicked()
+        }
+        ItemDelegate {
+            id: loreDelegate
+            width: menuDrawer.width
+            height: 64
+            topPadding: 21
+            bottomPadding: 21
+            text: qsTr("Lore")
+            Material.foreground: Palette.silverChalice
+            icon.source: "qrc:/assets/icons/lore.svg"
+            icon.color: Palette.silverChalice
+            onClicked: menuDrawer.loreClicked()
+        }
+        Rectangle {
+            width: menuDrawer.width
+            height: 1
+            color: Palette.white
+            opacity: 0.5
+        }
+        ItemDelegate {
+            id: languageItemDelegate
+            checkable: true
+            width: menuDrawer.width
+            height: 64
+            topPadding: 21
+            Material.foreground: Palette.silverChalice
+
+            contentItem: ColumnLayout {
+                id: content
+                spacing: 25
+                RowLayout {
+                    Layout.fillWidth: true
+                    implicitHeight: 64
+                    spacing: 16
+                    Image {
+                        source: "qrc:/assets/icons/language.svg"
+                    }
+                    Row {
+                        Layout.fillWidth: true
+                        Label {
+                            id: labelLanguage
+                            text: "Language - "
+                        }
+                        Label {
+                            text: TranslationsManager.currentLanguageText
+                            color: Palette.gallery
+                        }
+                    }
+
+                    Image {
+                        id: arrowImage
+                        source: "qrc:/assets/icons/arrow.svg"
+                    }
+                }
+
+                Loader {
+                    id: loader
+                    Layout.fillWidth: true
+                    sourceComponent: languagesRadio
+                    active: false
+                    opacity: 0
+                }
+
+                Component {
+                    id: languagesRadio
+                    Column {
+                        id: column
+                        spacing: 0
+                        padding: 0
+
+                        ButtonGroup {
+                            id: radioGroup
+                            buttons: column.children
+
+                            onCheckedButtonChanged: {
+                                radioGroup.checkedButton.text.color = Palette.gallery
+                                root.changed()
```

Comment:
What does this signal do? I see it's declared in `main.qml` but seems like it's never used, if it's not necessary just remove it.